llama_stack/distribution/routing_tables/vector_dbs.py

```diff
@@ -85,7 +85,7 @@ async def register_vector_db(
     async def unregister_vector_db(self, vector_db_id: str) -> None:
         existing_vector_db = await self.get_vector_db(vector_db_id)
         if existing_vector_db is None:
```

Comment:
this is redundant. get_vector_db returns VectorDB and throws otherwise.

Reply:
haha I beat you to it: https://github.com/meta-llama/llama-stack/pull/2980

But looks like your tests are passing. Confused why mine aren't.